### DISCUSSION TITLE
Use Taskfile instead of Rakefile

### DISCUSSION THREAD

**Discussion by uar316025** [Kraken-CI/kraken#304] [Ideas]
Hi, really impressed by work you done alone.
 
I would like to propose to migrate to https://taskfile.dev/
Main points:
- single dependency (only one executable)
- able to move common logic into separate files
- (in my opinion) simpler 

And I want to try to do it myself) just need your approval)

**Comment by godfryd** [comment]
Hello, thanks for proposal.
I switched from Makefile to Rakefile years ago.
I explored various other solutions.
What I like about Rake is that you program all in Ruby, regular decent language.
I could not find anything similar in all other solutions.
Rake allows for mixing easily regular code with shell commands via `sh` function.
Check what I'm doing e.g. in `task :github_release`. Things like that are hard without regular, imperative language.
Or in `task :publish_docker_aws`, `task :run_agent_in_lxd` or `at_exit`.

**Reply by uar316025** [reply]
As I can see, there should be no problem implementing that using taskfile,
I can make an example that implement tasks you listed above for the beginning

**Reply by godfryd** [reply]
ok, you could try that
